Sources/Quick/QuickSpec.swift

```diff
@@ -37,7 +37,21 @@ open class QuickSpec: QuickSpecBase {
     /// SwiftPM on macOS does not have the mechanism (test cases are automatically
     /// discovered powered by Objective-C runtime), so we needed the alternative
     /// way.
+    #if swift(>=4)
+    override open static var defaultTestSuite: XCTestSuite {
```

Comment:
This should be `class` instead of `static`.

https://developer.apple.com/documentation/xctest/xctestcase/1496289-defaulttestsuite#